launcher/modManager/cmodlist.cpp

```diff
@@ -92,6 +92,11 @@ bool CModEntry::isUpdateable() const
 	return false;
 }
 
+bool CModEntry::isEssential() const
```

Comment:
Could we unify naming? So it is either essential or stored locally?

Reply:
The intention is to check mod is it essential or not. I've considered 3 options:
* by name - the problem is that you may have one more mod in mods folder which you probably'd like to disable/delete if we update it from main repo
* by parameter - the problem here is that someone may include this parameter into other mods (intentionally or accidentally) so it'll look as a bug "cannot disable mod"
* by path - that looks like safest way from my pov. Anyway it's prohibited to delete/install mods locally from launcher. If someone store mods there manually, those should probably know what they are doing